### ISSUE TITLE
A warning in ProcessMenuBuilder.module

### ISSUE THREAD

**Issue by ivangretsky** [kongondo/MenuBuilder#44]
Good day!

A php warning  is shown in admin when viewing a kist of menus and one of them doesn't have items. 

```
Warning: count(): Parameter must be an array or an object that implements Countable in ***\site\modules\ProcessMenuBuilder\ProcessMenuBuilder.module on line 1129
```

**Comment by kongondo** [comment]
Thanks @ivangretsky . I'll sort it out, but it may be a while.

**Comment by kongondo** [comment]
Actually, this was already fixed on the dev branch in this commit f8b982f. This has now been merged into master. I'm closing this, thanks.
